engine/src/main/scala/cromwell/webservice/WebServiceUtils.scala

```diff
@@ -52,10 +71,10 @@ object WebServiceUtils extends WebServiceUtils {
   // AEN 2018-12-05
   implicit class EnhancedThrowable(val e: Throwable) extends AnyVal {
     def failRequest(statusCode: StatusCode, warnings: Seq[String] = Vector.empty): Route = {
-      completeResponse(statusCode, prettyPrint(APIResponse.fail(e)), warnings)
+      completeResponse(statusCode, ContentTypes.`application/json`, prettyPrint(APIResponse.fail(e)), warnings)
```

Comment:
It looks like the comment above might be out of date now - could you check and delete it if yes?

I have it on good authority that there are dragons in this area, but if you're getting the requisite thumbs that's good enough for me.

Reply:
( See conversation in #fix-cromwell on February 7 )

Reply:
I couldn't follow any outcome/decision of https://broadinstitute.slack.com/archives/GC6MN578V/p1549561615005300

Are there no longer:

> customers who rely on the pretty printing to display the error directly in a terminal

i.e. those who use `curl` but refuse to add-on `| jq .`? This diff just fixes the `Content-Type` header instead of leaving it [`text/plain(UTF-8)`](https://github.com/akka/akka-http/blob/v10.1.7/akka-http-core/src/main/scala/akka/http/scaladsl/model/HttpEntity.scala#L276), so the code still jumps through hoops to avoid using the akka-http marshallers.

Reply:
I don't wanna be a jerk and hold up your PR with FUD, but I think we have to assume those customers (i.e. FireCloud) still have this requirement, unless we can prove otherwise.

I vaguely remember a scenario where they display the error JSON in a web UI that does not itself have pretty-printing, so it has to be pretty-printed for them. If the combination of `prettyPrint()` and `ContentTypes.application/json` actually maintains the formatting, then that seems just fine.

Jeff knows the details and it looks like he's back Monday if you're patient.

Reply:
FYI- here's how a Chrome guest window renders what was `test/plain(UTF-8)` and is now `application/json`:

<img width="1045" alt="screen shot 2019-02-19 at 10 17 13 am" src="https://user-images.githubusercontent.com/791985/53025889-d7caa500-342f-11e9-96eb-01420e5eb18a.png">

Reply:
👍 thank you for checking!